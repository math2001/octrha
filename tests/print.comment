loads 109 in r0
prints it as a decimal (10) and on a new line, as a hex number (a), and after
that as a character (m)