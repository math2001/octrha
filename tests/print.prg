25 6d 06 ff 07 ff 05 ff