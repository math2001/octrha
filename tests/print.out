109
6d
m
